cupy/_sorting/search.py

```diff
@@ -139,6 +139,41 @@ def nonzero(a):
     _util.check_array(a, arg_name='a')
     return a.nonzero()
 
+def mask_indices(n, mask_func, k=0):
```

Comment:
Is this the right module for this function? I placed it here because of the similarities to  `nonzero`.

Reply:
Hi @shwina maybe this is the file to go:
https://github.com/cupy/cupy/blob/c0c5f9e4b7e8693ce6a435276d28e21ae3a55bf5/cupy/_indexing/generate.py#L435

Reply:
👍 Thanks!